src/main/resources/Tour/WebHome.xml

```diff
@@ -1,5 +1,4 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Note: this diff (about this document) actually completely removes the old page (with the demo tour it contained) to add a AWM home page. It lists all existing tours. That's why the diff is huge.